src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -43,7 +43,15 @@ void clang_c_languaget::build_compiler_args(const std::string &tmp_dir)
 {
   compiler_args.emplace_back("clang-tool");
 
-  compiler_args.push_back("-I" + tmp_dir);
+  const std::string *libc_headers = internal_libc_header_dir();
+  if(libc_headers)
+  {
+    compiler_args.push_back("-isystem");
+    compiler_args.push_back(*libc_headers);
+  }
+
+  compiler_args.push_back("-isystem");
```

Comment:
Same as line 49. pehaps make it common?

Reply:
We would have the same `if` plus another std::vector. I don't think it's worth it at this point. We don't even expose `-isystem` as a cmdline parameter at this point.